### PR TITLE
Added solution as an acceptable file type

### LINKED ISSUE #7835: Missing command-line option docs
You can use .sln file to build your projects, `dotnet build -h` also missing this

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 910cd365-d63a-00dc-417c-db017f83bf6f
* Version Independent ID: 5e495ddd-140e-de5c-a793-d4a9f88be51e
* Content: [dotnet build command - .NET Core CLI](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build?tabs=netcore2x)
* Content Source: [docs/core/tools/dotnet-build.md](https://github.com/dotnet/docs/blob/master/docs/core/tools/dotnet-build.md)
* Product: **dotnet-core**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

Clarified that you can use a solution instead of a project.

Fixes #7835

### CODE REVIEW

File: docs/core/tools/dotnet-build.md [hunk 1]
```diff
@@ -55,9 +57,9 @@ In order to produce a library, omit the `<OutputType>` property. The main differ
 
 ## Arguments
 
-`PROJECT`
+`PROJECT | SOLUTION`
 
-The project file to build. If a project file is not specified, MSBuild searches the current working directory for a file that has a file extension that ends in *proj* and uses that file.
+The project or solution file to build. If a project or solution file is not specified, MSBuild searches the current working directory for a file that has a file extension that ends in *proj* and uses that file.
```
Comment (rpetrusha) [inline docs/core/tools/dotnet-build.md]: or it uses a *sln* file

**Review by rpetrusha — APPROVED** [review]
This looks good, @Thraka. Just one thing to address: build looks either for a *sln* file or a project file ending in *proj*. Once you make that change, you can merge.

File: docs/core/tools/dotnet-build.md (hunk 1 above)
Comment (pkulikov) [reply docs/core/tools/dotnet-build.md]: @rpetrusha should it be specified what the build looks for at first? In other words, if both *proj* and *sln* files present in the folder, which one is picked up by `dotnet build`?

Comment (Thraka) [reply docs/core/tools/dotnet-build.md]: @pkulikov Neither is picked up. If there are more than one it errs and tells you to specify which you want to use.

**Comment by Thraka** [comment]
@rpetrusha Good catch. I added that.

**Pull request merged**